code/modules/photography/camera/camera.dm

```diff
@@ -212,6 +223,8 @@
 	after_picture(user, P, flag)
 	blending = FALSE
 
+/obj/item/camera/proc/checks_detectable(atom/A)
+	return is_type_in_typecache(A, detectable_invisible_atom)
```

Comment:
> this proc is pretty useless now tbh

since the if logic is difficult to read, this is better than using checking_type_cache directly. the proc name is changed now.